IntelliTect.Analyzer/IntelliTect.Analyzer.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -106,14 +117,7 @@ private static void VerifyDiagnosticResults(IEnumerable<Diagnostic> actualResult
         {
             int expectedCount = expectedResults.Length;
             int actualCount = actualResults.Count();
-
-            if (expectedCount != actualCount)
-            {
-                string diagnosticsOutput = actualResults.Any() ? FormatDiagnostics(analyzer, actualResults.ToArray()) : "    NONE.";
-
-                Assert.IsTrue(false,
-                    string.Format("Mismatch between number of diagnostics returned, expected \"{0}\" actual \"{1}\"\r\n\r\nDiagnostics:\r\n{2}\r\n", expectedCount, actualCount, diagnosticsOutput));
-            }
+            Assert.AreEqual(expectedCount, actualCount, $"Mismatch between number of diagnostics returned, expected \" {expectedCount} \" actual \" {actualCount} \"\r\n\r\nDiagnostics:\r\n{(actualResults.Any() ? FormatDiagnostics(analyzer, actualResults.ToArray()) : "NONE.")}\r\n");
```

Comment:
Should this be using `Environment.NewLine` rather than `"\r\n"`?

Reply:
I'll update them to use Environment.NewLine